src/control/lib/spdk/src/nvme_control.c

```diff
@@ -28,10 +28,16 @@
 #include "nvme_control.h"
 #include "nvme_control_common.h"
 
-struct wipe_sequence {
-	struct ns_entry	*ns_entry;
-	char		*buf;
-	int		 is_completed;
+enum lba0_write_result {
+	LBA0_WRITE_PENDING	= 0x0,
+	LBA0_WRITE_SUCCESS	= 0x1,
+	LBA0_WRITE_FAIL		= 0x2,
+};
+
+struct lba0_data {
+	struct ns_entry		*ns_entry;
+	char			*buf;
+	enum lba0_write_result	 was_written;
```

Comment:
nit - the name still implies a boolean. write_result or write_status may be a better name.